mlrun/datastore/base.py

```diff
@@ -121,7 +122,18 @@ def download(self, key, target_path):
     def upload(self, key, src_path):
         pass
 
-    def as_df(self, url, subpath, columns=None, df_module=None, format="", **kwargs):
+    def as_df(
+        self,
+        url,
+        subpath,
+        columns=None,
+        df_module=None,
+        format="",
+        start_time=None,
+        end_time=None,
+        filter_column=None,
```

Comment:
You're calling it `filter` as if the only filter here is time, but it might not be true, I suggest `time_column` (throughout all the APIs)